dex/farm/src/claim_boost_only.rs

```diff
@@ -37,12 +35,12 @@ pub trait ClaimBoostOnlyModule:
     #[endpoint(claimBoostedRewards)]
     fn claim_boosted_rewards(
         &self,
-        opt_user: OptionalValue<ManagedAddress>
+        opt_user: OptionalValue<ManagedAddress>,
     ) -> EsdtTokenPayment<Self::Api> {
         let user = match opt_user {
```

Comment:
This is actually worse than before, since you read the entire struct twice still. Read only once at the beginning of this function.

Reply:
Fixed